README.rst

```diff
@@ -226,6 +226,29 @@ Choose from the list of available rules:
    | Replaces dirname(__FILE__) expression with equivalent __DIR__ constant.
    | *Rule is: risky.*
 
+* **doctrine_annotation_argument_space**
```

Comment:
question: would any of this fixers run before/after another ?
hint: make an integration test using all doctrine_* fixers to see they work as expected when one use all of them